argm.c

```diff
@@ -330,7 +338,6 @@ argm_combine_universal(PG_FUNCTION_ARGS, int32 compareFunctionResultToAdvance)
     {
         int preference = 0;
         int i;
-        state0 = (ArgmState *) PG_GETARG_POINTER(0);
```

Comment:
This was the other part of the bug: once we get a zero pointer here instead of an SQL null, `PG_ARGISNULL(0)` is false, so we get to this branch, and attempt to use a zero pointer.